src/Config/Authentication.cs

```diff
@@ -1,59 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-namespace Azure.DataApiBuilder.Config
-{
-    /// <summary>
-    /// Authentication configuration.
-    /// </summary>
-    /// <param name="Provider">Identity Provider. Default is StaticWebApps.
```

Comment:
Please preserve the comments of the record members. This is useful for context. I notice the corresponding record `AuthenticationOptions.cs` is missing these comments. Similar to this, please identify all other situations where the old file is being deleted, and move the remaining relevant comments to the corresponding new Object Model.

Reply:
Went through the old object model and ported over the comments that mapped, and expanded on others that are ambiguous.

I didn't bring some things in that provided really limited value (such as annotating an enum stating it's an enum).